### PR TITLE
Create more language pages

### PR DESCRIPTION
**Pull request by cefoo**
# Create more language pages

Fixes # 304

## Type of PR

- Edits _languages.yml_ and creates new language pages

### Checklist:

- [x] I have read the [contributing guidelines](contributing.md).
- [x] I have followed the [style guide](http://machinetranslate.org/style).

### CODE REVIEW

File: _data/languages.yml [hunk 1]
```diff
@@ -241,7 +241,7 @@
   territories: [ ro, ru, ua ]
 
 -
-  codes: [ cnr ]
+  codes: [ cnr, me ]
```
Comment (bittlingmayer) [inline _data/languages.yml:244]: I'd remove this, it's a Niutrans mistake afaict, our infra will handle it anyway.

File: _data/languages.yml [hunk 2]
```diff
@@ -645,7 +677,7 @@
   territories: [ id ]
 
 -
-  codes: [ ka, kat, geo ]
+  codes: [ ka, kat, geo, jy ]
```
Comment (bittlingmayer) [inline _data/languages.yml:680]: ditto

File: _data/languages.yml [hunk 3]
```diff
@@ -859,7 +921,7 @@
   territories: [ in ]
 
 -
-  codes: [ mn, mon ]
+  codes: [ mn, mon, mo ]
```
Comment (bittlingmayer) [inline _data/languages.yml:924]: ditto

Comment (bittlingmayer) [reply _data/languages.yml:924]: The issue is that in other contexts this is used for Moldovan.

Comment (cefoo) [reply _data/languages.yml]: done

File: _data/languages.yml (hunk 2 above)
Comment (cefoo) [reply _data/languages.yml]: done

File: _data/languages.yml (hunk 1 above)
Comment (cefoo) [reply _data/languages.yml]: done

**Review by bittlingmayer — APPROVED** [review]

**Pull request merged**